src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,30 +40,40 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${todo?.id}`}
             </div>
 
             {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
               className="delete"
               data-cy="modal-close"
+              onClick={() => {
+                callbackTodo(null);
+                setUser(null);
+              }}
             />
           </header>
 
           <div className="modal-card-body">
             <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
+              {todo?.title}
             </p>
 
             <p className="block" data-cy="modal-user">
               {/* <strong className="has-text-success">Done</strong> */}
-              <strong className="has-text-danger">Planned</strong>
+              <strong className={classNames(
+                { 'has-text-danger': !todo?.completed },
+                { 'has-text-success': todo?.completed },
+              )}
+              >
+                {todo?.completed ? 'Done' : 'Planned'}
+              </strong>
```

Comment:
```suggestion
             {foundTodo?.completed
                ? <strong className="has-text-success">Done</strong>
                : <strong className="has-text-danger">Planned</strong>}
```

You can simplify conditional rendering a little in order to not duplicate ternaries for completed todos